packages/eslint-plugin/src/utils/getAllPackageInfo.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs-extra');
+const path = require('path');
+const lernaAlias = require('lerna-alias');
+const { findGitRoot } = require('./configHelpers');
+
+/**
+ *  @typedef {{name: string, version: string, dependencies: {[key: string]: string}}} PackageJson
+ *  @typedef {{packagePath: string, packageJson: PackageJson}} PackageInfo
+ *  @typedef {{[packageName: string]: PackageInfo}} AllPackageInfo
+ */
+
+function getAllPackageInfo() {
```

Comment:
what's different about this `getAllPackageInfo` compared to this one ? 

https://github.com/microsoft/fluentui/blob/master/scripts/monorepo/getAllPackageInfo.js